pnpm-lock.yaml

```diff
@@ -3660,6 +3660,15 @@ packages:
       marked: 0.3.19
     dev: false
 
+  /@cloudflare/workerd-darwin-64@1.20231025.0:
```

Comment:
These changes are caused by updating package.json files after running `pnpm i` on your local machine. (i.e. they are only added during the post build stuff I think). We should probably avoid that if possible.